fc/auth/envvarauth.py

```diff
@@ -0,0 +1,16 @@
+from .auth import Auth
+import os
+
+
+class EnvVarAuth(Auth):
+    def __init__(self, username=None):
```

Comment:
This auth class doesn't need a `username`.  In fact, I'm thinking the `KeyboardAuth` class doesn't even need it since, if the username is specified, it will be returned in the `ComboAuth.username` method.

I you could remove it from this class, I can remove it from the `KeyboardAuth` class.

Reply:
OK, well I removed the `__init__` function altogether as I guess it is not needed, right?